### ISSUE TITLE
stuck on heap spray, Ryzen 3700x, x470

### ISSUE THREAD

**Issue by acharya8848** [checkra1n/BugTracker#848]
**Tell us about your setup:**
1. What iDevice are you using? iPhone 6s+
2. On what version of iOS is it? 13.3
3. What version of checkra1n are you using? GUI or CLI? 0.9.8 CLI and GUI, tried both
4. What is your host system (OS version? Hackintosh? VM? etc.)? Ubuntu 19.10, everything up to date
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)? OEM type A to lightning cable

**What are the steps to reproduce the issue?**
No specific steps to reproduce the problem for me. Just trying to jailbreak gets stuck on heap spray. Moves forward to exploiting for a moment and then resets to the initial step again.

**What do you expect, and what is happening instead?**
I expected my device to boot up in a jailbroken state, but the device just stays in DFU mode.

**Does the issue also occur if you enable Safe Mode?**
I didn't try safe mode. Instead, I tried again on another computer, a laptop with an Intel processor, and it worked. 

**Any other info, error logs, screenshots, ...?**
No errors reported by the program, it just kept on looping the steps.

**Comment by truff1ep1g** [comment]
Please try this 

https://github.com/foxlet/bootra1n/blob/master/README.md

**Comment by acharya8848** [comment]
Nope, it doesn't work. Just stuck at the very beginning of the process.

**Comment by truff1ep1g** [comment]
Wow. Sorry.

**Comment by nergzd723** [comment]
@acharya8848 
The only solution now is ra1nUSB and older version of checkra1n.

**Comment by truff1ep1g** [comment]
﻿Why is that. I haven’t been really looking at any new posts or feeds the last few days. What’s happened. ?
Did they pull support for checkra1n or something

**Comment by acharya8848** [comment]
I have tried ra1nusb. Neither did it work on my amd machine nor did it on my intel laptop.

**Comment by truff1ep1g** [comment]
That sux.   Sorry i couldn’t help more.

**Comment by truff1ep1g** [comment]
Try it in ram mode. The second option.?  Maybe that’ll work for you. Are u using the port labeled “ss” or with blue inside the connection ?  To plug the iPhone cable in to.  Put the flash drive into one of the others. Also turn your passcode on the phone off sometimes that does help
